bin/zerobufCxx.py

```diff
@@ -958,8 +962,12 @@ def introspection_functions(self):
         functions = []
         functions.append(Function("std::string", "getTypeName() const final",
                                   "return \"{0}\";".format(zerobufName), split=False))
```

Comment:
You can make these lines more readable if you combine " and '

    'return "{0}";'.format(zerobufName)

I also find the version below even more readable, but that's a matter of taste:

    'return "%s";' % zerobufName